README.md

```diff
@@ -109,22 +109,48 @@ direct you to a page that gives you the ability to submit a request to the
 master repository to merge in the changes you committed.
 
 ##Style Guide
+### Expected Values
+
+When a value is expected or described within plain text, but isn't specifically code, single quotes should be used.
+In situations where it is important to group text to be specific to the concept, single quotes should also be used.
```

Comment:
I can see what you mean because of the example, but the wording here isn't clear. That said, this is the style guide so let's not worry too much!!!